src/styles/header.scss

```
@@ -0,0 +1,89 @@
+@import './variables';
+
+html {
+  font-family: Roboto, Raleway, sans-serif;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  max-width: 100%;
+  flex-shrink: 0;
+  background: #fff;
+
+  &__logo {
+    display: flex;
+    align-items: center;
+
+    &-img {
+      width: 40px;
+      height: 40px;
+      background: lightgray 50% / cover no-repeat;
+    }
+  }
+
+  &__nav {
+    display: flex;
+
+    &-list {
+      display: flex;
+      justify-content: space-between;
+      list-style: none;
+      padding: 0;
+      margin: 0;
+      height: 60px;
+      line-height: 60px;
+    }
+
+    &-item {
+      margin-right: 20px;
+
+      &:last-child {
+        margin-right: 0;
+      }
+    }
+
+    &-link {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      font-size: 12px;
+      padding: 0;
+      margin: 0;
+      font-style: normal;
+      color: #060b35;
+      font-weight: 500;
+      text-decoration: none;
+      text-transform: uppercase;
+      transition: color 300ms;
+
+      &:hover {
+        color: #00acdc;
+      }
+    }
+  }
+}
+
+.is-active {
+  position: relative;
+  color: $main-blue-accent-color;
+
+  &::after {
+    content: '';
+    position: absolute;
+    bottom: 0;
+    width: 37px;
```

Don't set `width` explicitly.

You need to be able to re-use the same style on the other menu elements with different widths.

You can try following (as long as your `after` is positioned absolutely):
```suggestion
    left: 0;
    right: 0;
```